WooCommerce/Classes/ViewRelated/Dashboard/Analytics Hub/StatsCard.swift

```diff
@@ -0,0 +1,99 @@
+import SwiftUI
+
+/// Resuable report card made for the Analytics Hub.
+///
+struct AnalyticsReportCard: View {
+
+    let title: String
+    let leadingTitle: String
+    let leadingValue: String
+    let leadingGrowth: String
+    let leadingGrowthColor: UIColor
+    let trailingTitle: String
+    let trailingValue: String
+    let trailingGrowth: String
+    let trailingGrowthColor: UIColor
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: Layout.titleSpacing) {
+
+            Text(title)
+                .foregroundColor(Color(.text))
+                .footnoteStyle()
+
+            HStack {
+
+                /// Leading Column
+                ///
+                VStack(alignment: .leading, spacing: Layout.salesColumnSpacing) {
+
+                    Text(leadingTitle)
+                        .calloutStyle()
+
+                    Text(leadingValue)
+                        .titleStyle()
+
+                    GrowthTag(value: leadingGrowth, backgroundColor: leadingGrowthColor)
+
+                }
+                .frame(maxWidth: .infinity, alignment: .leading)
+
+                /// Net Sales
+                ///
+                VStack(alignment: .leading, spacing: Layout.salesColumnSpacing) {
+                    Text(trailingTitle)
+                        .calloutStyle()
+
+                    Text(trailingValue)
+                        .titleStyle()
+
+                    GrowthTag(value: trailingGrowth, backgroundColor: trailingGrowthColor)
+                }
+                .frame(maxWidth: .infinity, alignment: .leading)
+            }
+        }
+        .padding(Layout.cardPadding)
+    }
+}
+
+private struct GrowthTag: View {
```

Comment:
Just a naming suggestion: What do you think of using the term `Delta` or `DeltaPercentage` instead of `Growth` here? I think both names are valid, but we're already using the term "delta" on the Android side, so it might be helpful to use consistent naming for the sake of discussion.

Reply:
Sure, changed it!